plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2247,6 +2247,8 @@ elementgui.keybind.key_procedure_triggers=Key binding procedure triggers
 elementgui.keybind.error_key_needs_name=Key binding needs a name
 elementgui.keybind.error_key_category_needs_name=Key binding category key name
 elementgui.living_entity.condition_solid_bounding_box=Solid bounding box condition
+elementgui.living_entity.visual_scale=Entity Visual Scale:
```

Comment:
We don't use this type of capitalization in UIs, lowercase 2nd and 3rd word, same for bbox scale